docs/apps/interchain-accounts/client.md

```diff
@@ -0,0 +1,232 @@
+<!--
+order: 7
+-->
+
+# Client
+
+## CLI
+
+A user can query and interact with the Interchain Accounts module using the CLI.
+
+### Controller
+
+A user can query and interact with the controller submodule.
+
+#### Query
+
+The `query` commands allow users to query the controller submodule.
+
+```shell
+simd query interchain-accounts controller --help
+```
+
+#### `interchain-account`
+
+The `interchain-account` command allows users to query the controller submodule for the interchain account address for a given owner on a particular connection.
+
+```shell
+simd query interchain-accounts controller interchain-account [owner] [connection-id]
+```
+
+Example:
+
+```shell
+simd query interchain-accounts controller interchain-account cosmos1.. connection-0
```

Comment:
nit: should we use a full bech32 address in examples?

Reply:
I don't have a strong preference. I used this because I saw it in the SDK docs and I thought it removed a bit of noise, but happy to add the full address if people think it's better.